charactersheet/charactersheet/index.html

```diff
@@ -73,100 +73,131 @@ <h2 data-bind='text: playerTitle'></h2>
+		<li role="presentation" data-bind="click: activatePartyTab, css: partyTabStatus"><a href="#party" aria-controls="party" role="tab" data-toggle="tab">Party Chat</a></li>
 	</ul>
 	<div class="tab-content">
-		<div role="tabpanel" class="tab-pane" id="character" data-bind="with: characterTabViewModel, css: characterTabStatus">
+		<div role="tabpanel" class="tab-pane" id="profile" data-bind="with: profileTabViewModel, css: profileTabStatus">
 			<div class="row">
 				<div class="col-md-7 col-sm-12 col-padded">
 					<div data-bind="template: { name: 'profile.tmpl', templateUrl: 'profile', data: profileViewModel }">
 					</div>
 				</div>
-				<div class="col-md-5 col-sm-6 col-padded">
+				<div class="col-md-5 col-sm-12 col-padded">
 					<div data-bind="template: { name: 'appearance.tmpl', templateUrl: 'appearance', data: appearanceViewModel }">
 					</div>
 				</div>
-
-				<div class="col-md-4 col-sm-6 col-padded">
-					<div data-bind="template: { name: 'ability.tmpl', templateUrl: 'ability_scores', data: abilityScoresViewModel }">
+			</div>
+			<div class="row">
+				<div class="col-md-12 col-sm-12 col-padded">
+					<div data-bind="template: { name: 'feat_traits.tmpl', templateUrl: 'feat_traits', data: featuresTraitsViewModel }">
 					</div>
 				</div>
-				<div class="col-md-4 col-sm-6 col-padded">
-					<div data-bind="template: { name: 'feats_prof.tmpl', templateUrl: 'feats_prof', data: featsProfViewModel }">
-					</div>
+			</div>
+		</div>
+<!-- End Profile Tab -->
+	<div role="tabpanel" class="tab-pane" id="stats" data-bind="with: statsTabViewModel, css: statsTabStatus">
+		<div class="row">
+			<div class="col-md-12">
+				<div id="stats" data-bind="template: { name: 'stats.tmpl', templateUrl: 'stats', data: statsViewModel }">
 				</div>
-				<div class="col-md-4 col-sm-6 col-padded">
-					<div data-bind="template: { name: 'feat_traits.tmpl', templateUrl: 'feat_traits', data: featuresTraitsViewModel }">
-					</div>
+			</div>
+		</div>
+		<div class="row">
+			<div class="col-md-4 col-sm-12 col-padded">
+				<div data-bind="template: { name: 'ability.tmpl', templateUrl: 'ability_scores', data: abilityScoresViewModel }">
 				</div>
 			</div>
-			<img src="/images/heart.png" height="35" style="margin-top:-12px;" />&nbsp;
-			<h3 style="display:inline;">
-				Health &amp; Stats
-			</h3>
-      <hr />
-			<div class="row">
-				<div class="col-md-8">
-					<div id="stats" data-bind="template: { name: 'stats.tmpl', templateUrl: 'stats', data: statsViewModel }">
-					</div>
+			<div class="col-md-4 col-sm-12 col-padded">
+				<div id="spell_stats" data-bind="template: { name: 'spell_stats.tmpl', templateUrl: 'spell_stats', data: spellStatsViewModel }">
 				</div>
-				<div class="col-md-4">
-					<div id="spell_stats" data-bind="template: { name: 'spell_stats.tmpl', templateUrl: 'spell_stats', data: spellStatsViewModel }">
-					</div>
+			</div>
+			<div class="col-md-4 col-sm-12 col-padded">
```

Comment:
Could you fix the saving throws? Right now they're not in a panel like they should be. From what I can see, the panel is either never opened or closed from the previous panel. 
![screenshot 2016-02-03 09 58 26](https://cloud.githubusercontent.com/assets/3310280/12791634/e7f53bce-ca5c-11e5-9cb3-0e2787385ff5.png)